globalvars.py

```diff
@@ -129,9 +128,66 @@ class GlobalVars:
 
     api_request_lock = threading.Lock()
 
-    num_posts_scanned = 0
-    post_scan_time = 0
-    posts_scan_stats_lock = threading.Lock()
+    class PostScanStat:
+        """ Tracking post scanning data """
+        # Last change was on 19 May 2020.
```

Comment:
Let's not have comments which state the last change date. While knowing that information is helpful, such comments commonly become out of date. Even when such a comment exists, one has to go look at the git blame or git history to actually know if you can believe such a comment.

OTOH, it's not unreasonable to leave a comment that something changed on a date and why, if it's quite relevant to why the code is doing what it's doing.

Reply:
The hope is that this comment will be updated every time one changes code logic in the section. By doing so, we can tell if the documentation is out of sync (there is a similar comment on every section of the doc, and hopefully one will update the doc if he/she performed a major change (i.e. logic change) in code)

Reply:
While I understand the desire, wanting to know when something was changed is one of the reasons we have source code control systems, which maintain this type of information automatically.

The realty is that you can almost never trust that such a comment is accurate. Such comments, unless automatically maintained, will never be 100% accurate. The only way you know if it's accurate is to go to the `git` record and see when the code was last actually modified, which is exactly what you would have had to do to get the information when the comment isn't there. So, when you see such a comment, you can either ignore it, verify it for the same effort it would take to find the accurate information, or trust it and potentially have the wrong information.